broadcaster/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@axiom-crypto/broadcaster",
+  "version": "2.0.5",
+  "author": "Intrinsic Technologies",
+  "license": "MIT",
+  "description": "Broadcaster SDK for broadcasting Axiom data between chains.",
+  "main": "js/index.js",
+  "types": "index.d.ts",
+  "browser": "web/index.js",
+  "scripts": {
+    "build": "rm -rf ./dist/* && tsc && node scripts/postTsc.js",
+    "test": "jest"
+  },
+  "publishConfig": {
+    "directory": "dist"
+  },
+  "keywords": [
+    "axiom",
+    "ethereum",
+    "zero knowledge",
+    "zk",
+    "coprocessor",
+    "crypto"
+  ],
+  "dependencies": {
+    "@axiom-crypto/client": "link:../client/dist",
```

Comment:
We should remove these dependencies: we want `broadcaster` to be a separate package that does not depend on `client` or `core`